.github/workflows/test.yaml

```diff
@@ -43,6 +43,7 @@ jobs:
     strategy:
       matrix:
         go_version:
+        - '1.20'
         - '1.19'
         - '1.18'
```

Comment:
Please drop the oldest release here, we just test against the current release (handled above on line 27) and the two previous releases.